active/community.md

```diff
@@ -0,0 +1,75 @@
+# WG Charter Template
+
+## Scope of responsibilities
+
+This group exists to support the official community platforms and those in elevated roles on those platforms.
+
+The Forum and the Discord are core parts of the overall community and those that help and moderate these platforms take significant time from their day to help others. We can support them through training and taking action to make their lives easier.
+
+The broad goals of the group are:
+  - facilitate and centralise discussion between moderators and admins on the various platforms
+  - coordinate efforts for training and processes around moderation
+  - provide support to improve the platforms to lighten moderation and admin workloads
+  - provide training and support to the wider community on how to use the platforms effectively
+
+- What actions are you proposing the WG be allowed to take directly?
+  - Creation and management of processes related to community platforms.
+    - Adding/removing moderators & other roles
+    - Training for moderators
+    - Ensuring moderation is consistent across teams and platforms
+  - Taking ideas from the community to improve the community spaces.
+- Which actions will the WG take back to the Board for votes?
+  - Moving platforms if required
+  - Budget to improve existing platforms
+
+There is a lot of potential ideas in this space. Part of the WG will be setting up a process to take ideas and have a backlog of these ideas.
+
+### Delegated responsibilities
+
+With regards to Django Software Foundation responsibilities and resources, the group operates with:
+
+- I'm unsure what power's might be delegated, if any. Board members would need to input into this.
+- Admin privileges to members of the forum & Discord or existing admin representive joining as a member.
+
+## Initial membership
+
+- Chair: TBD
+- Co-Chair: TBD
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBD
+- Other members:
+  - Andrew Miller
+
+## Future membership
+
+Answer the following questions here:
+
+- Who is eligible to join? Any volunteer, or are there specific requirements?
+  - At least a DSF member, ideally already holding an elevated role in the forum or Discord (or other community group)
+- How do people who want to join sign up / volunteer / express interest?
+  - Message of the existing group
+- How will decisions on adding/removing members be handled?
+  - Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members; the WG will directly vote on new Chair/Co-Chairs.
```

Comment:
One thing I suggest you think about here is how you will remove inactive members, otherwise you might end up in a situation where you can never get 50%+1.

For the a11y team we have a system where you have to specifically opt in each year or you get removed by default, though we only had a big enough team to require this very recently so this hasn't been put into practice yet.

Reply:
+1. The common pattern we seem to be adopting across WGs is a yearly re-affirmation that you still want to be part of the group, giving people a convenient time to step away. It feels like a good pattern.